src/main/webapp/pages/template/main-navigation.html

```diff
@@ -37,10 +37,6 @@
                             </a>
                         </li>
                         <li role="separator" class="divider"></li>
-                        <li>
-                            <a th:href="@{/projects/new}" th:text="#{nav.main.project-create}">
```

Comment:
Remove this string from the messages file

Reply:
Removed in 1097b0216a8f5a7dc3acb50adc0b77878cf08a46